politeiawww/database/database.go

```diff
@@ -146,6 +152,8 @@ type Database interface {
 	UserUpdate(User) error                   // Update existing user
 	AllUsers(callbackFn func(u *User)) error // Iterate all users
 
+	ProposalAccessTimeNew(string, string) error                        // add a new proposal access time record
```

Comment:
Why do we need a whole new record for access times?  Wouldn't adding the access times onto the user object be simpler?